test/integration/mount_start_test.go

```diff
@@ -30,6 +31,9 @@ func TestMountStart(t *testing.T) {
 	if NoneDriver() {
 		t.Skip("skipping: none driver does not support mount")
 	}
+	if KicDriver() && runtime.GOOS == "windows" {
+		t.Skip("skipping: Docker driver on Windows fails mounting due to 'The notification platform is unavailable'")
```

Comment:
I suggest one of these


- add a TODO github issue in the t.Skip for when we fix this on the user side 
- or let the user know it wont work in the UI if they start minikube on windows that option

Reply:
Done